src/reactviews/pages/ConnectionDialog/components/advancedOptionsDrawer.component.tsx

```diff
@@ -35,11 +37,37 @@ export const AdvancedOptionsDrawer = ({
     setIsAdvancedDrawerOpen: React.Dispatch<React.SetStateAction<boolean>>;
 }) => {
     const context = useContext(ConnectionDialogContext);
+    const [searchSettingsText, setSearchSettingText] = useState<string>("");
+    const [userOpenedSections, setUserOpenedSections] = useState<string[]>(["General"]);
 
     if (context === undefined) {
         return undefined;
     }
 
+    const doesGroupHaveVisibleOptions = (group: ConnectionComponentGroup) => {
```

Comment:
Preference to favor `function myFunc(arg) {}` over `const myFunc = (arg) => {}` for readability.